BeehiveCommunity/Frontend_Mentor
# Fixed Background Image Visibility Issue on Profile Card (#31)

## What does this PR do?

This PR fixes the issue where the background images (`bg-top` and `bg-bottom`) on the profile card disappear or become less visible on smaller screens. The issue is addressed by resizing the background images using `vw` units and adjusting their position to ensure visibility across different viewport sizes.

Fixes #31


## Type of change


## How should this be tested?

1. Open the profile card on various screen sizes (desktop, tablet, mobile).
2. Verify that the background images remain visible and proportionate as the screen size reduces.
3. Check for any visual regressions on larger screens.

- [X] Test A: Check visibility of `bg-top` and `bg-bottom` images on a mobile screen (max-width: 375px).
- [X] Test B: Validate responsiveness and layout on a tablet (max-width: 768px).

## Mandatory Tasks
